composer.lock

```diff
@@ -246,6 +246,60 @@
     ],
     "packages-dev": [
         {
+            "name": "doctrine/instantiator",
```

Comment:
I see quite a lot of new dependencies being added to `composer.lock`. Are those new dependencies introduced by phpunit 4.8, or maybe they are local to your installation, somehow?